build.gradle

```diff
@@ -7,15 +7,21 @@ buildscript {
         springBootVersion = '2.4.5'
         blackDuckCommonVersion = '59.0.4'
         junitPlatformCustomTestTags = 'docker, battery'
+        javaTargetCompatibility = 8
```

Comment:
The distinction between source and target is not clear. We need to be able to run on Java 8 systems at a minimum

Reply:
I also put the these variables in both the `root/bldsrc/build.gradle` and the `root/build.gradle` which I'm not confident is correct.